### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by slevenick**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+* **New Resource:** `google_apigee_envgroup_attachment` ([#8853](https://github.com/hashicorp/terraform-provider-google/pull/8853))
+
+IMPROVEMENTS:
+* bigquery: added `require_partition_filter` field to `google_bigquery_table` when provisioning `hive_partitioning_options` ([#8775](https://github.com/hashicorp/terraform-provider-google/pull/8775))
+* compute: added field `maintenance_window.start_time` to `google_compute_node_group` ([#8847](https://github.com/hashicorp/terraform-provider-google/pull/8847))
+* compute: added gVNIC support for `google_compute_instance_template` ([#8842](https://github.com/hashicorp/terraform-provider-google/pull/8842))
+* datacatalog: added `description` field to `google_data_catalog_tag_template ` resource ([#8851](https://github.com/hashicorp/terraform-provider-google/pull/8851))
+* tags: promoted `google_tags_tag_binding` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_key_iam_*` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
```
Comment (rileykarson) [inline CHANGELOG.md:18]:
```suggestion
```

Since this is first released in this PR anyways, we don't need to note this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ## 3.64.0 (Unreleased)
 
 FEATURES:
-* **New Resource:** `google_tags_tag_binding` (beta) ([#8840](https://github.com/hashicorp/terraform-provider-google/pull/8840))
```
Comment (melinath) [inline CHANGELOG.md]: This was added as beta-only but was also moved to GA this release (see the IMPROVEMENTS section below). The same is true for all the `google_tags_tag_key_iam_*` resources. I'm not sure how you want to handle that - maybe keep all of them as new features and not mention they were technically promoted?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+* **New Resource:** `google_tags_tag_value_iam_policy` ([#8844](https://github.com/hashicorp/terraform-provider-google/pull/8844))
+* **New Resource:** `google_apigee_envgroup_attachment` ([#8853](https://github.com/hashicorp/terraform-provider-google/pull/8853))
+
+IMPROVEMENTS:
+* bigquery: added `require_partition_filter` field to `google_bigquery_table` when provisioning `hive_partitioning_options` ([#8775](https://github.com/hashicorp/terraform-provider-google/pull/8775))
+* compute: added field `maintenance_window.start_time` to `google_compute_node_group` ([#8847](https://github.com/hashicorp/terraform-provider-google/pull/8847))
+* compute: added gVNIC support for `google_compute_instance_template` ([#8842](https://github.com/hashicorp/terraform-provider-google/pull/8842))
+* datacatalog: added `description` field to `google_data_catalog_tag_template ` resource ([#8851](https://github.com/hashicorp/terraform-provider-google/pull/8851))
+* tags: promoted `google_tags_tag_binding` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
```
Comment (rileykarson) [inline CHANGELOG.md:17]: Should we message these as new resources to TPG? We've done both formats historically.

File: CHANGELOG.md (hunk 1 above)
Comment (rileykarson) [reply CHANGELOG.md:18]: (same for the resources below for both of these)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+* tags: promoted `google_tags_tag_key_iam_*` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_key` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_value_iam_*` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_value` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+
+BUG FIXES:
+* compute: reverted datatype change for `mtu` in `google_compute_interconnect_attachment` as it was incompatible with existing state representation ([#8829](https://github.com/hashicorp/terraform-provider-google/pull/8829))
+* iam: fixed issue with principle and principleSet members not retaining their casing ([#8860](https://github.com/hashicorp/terraform-provider-google/pull/8860))
+* storage: fixed intermittent error in `google_storage_hmac_key` ([#8817](https://github.com/hashicorp/terraform-provider-google/pull/8817))
```
Comment (rileykarson) [inline CHANGELOG.md:26]: nit: more a question for the original PR, but what error?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,31 @@
...
+* datacatalog: added `description` field to `google_data_catalog_tag_template ` resource ([#8851](https://github.com/hashicorp/terraform-provider-google/pull/8851))
+* tags: promoted `google_tags_tag_binding` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_key_iam_*` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_key` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_value_iam_*` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+* tags: promoted `google_tags_tag_value` to GA ([#8854](https://github.com/hashicorp/terraform-provider-google/pull/8854))
+
+BUG FIXES:
+* compute: reverted datatype change for `mtu` in `google_compute_interconnect_attachment` as it was incompatible with existing state representation ([#8829](https://github.com/hashicorp/terraform-provider-google/pull/8829))
+* iam: fixed issue with principle and principleSet members not retaining their casing ([#8860](https://github.com/hashicorp/terraform-provider-google/pull/8860))
```
Comment (rileykarson) [inline CHANGELOG.md:25]: nit: Is it worth mentioning 3P identities here? On one hand, we fixed a bug. On the other, we support 3P identities by fixing it.

**Review by rileykarson — APPROVED** [review]

**Pull request merged**